packages/strapi-plugin-content-manager/admin/src/components/SortWrapper/index.js

```diff
@@ -1,11 +1,11 @@
 import styled from 'styled-components';
 
 const SortWrapper = styled.div`
-  margin-top: 7px;
-  margin-bottom: 10px;
-  padding-top: 11px;
-  border: 1px dashed #e3e9f3;
-  border-radius: 2px;
+  // &:after {
+  //   content: '';
+  //   clear: both;
+  // display: table;
+  // }
 `;
```

Comment:
Comments can be removed? Or `SortWrapper` could even be deleted and replaced by a simple div.